egs2/voice_assistant_demo/sds1/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
.gitignore requires to add entry 
`egs2/**/**/hub`

Reply:
Thanks for catching this! 
I updated .gitignore based on your suggestion.